### DISCUSSION TITLE
change from student to teacher

### DISCUSSION THREAD

**Discussion by zohim** [community/community#42862] [GitHub Education]
I was approved as a student, please can you help me reverse to teacher approval?
Thanks

**Comment by DSDmark** [comment]
Hi @zohim,

To apply for a teacher account on GitHub, you will need to do the following:

  -  Visit the GitHub Education website and click on the "Apply now" button.

  -  Follow the prompts to complete the application form. You will need to provide information about your educational institution and your role as a teacher.

  -  Submit the form and wait for a response from GitHub. They will review your application and determine whether you are eligible for a teacher account.

**Reply by zohim** [reply]
Thank you @DSDmark , 
I already completed the approval process, but instead of choosing teacher option, I selected the student. I just need to be reversed to the teacher status. GitHub does not allow a second approval submission with my ID.

**Reply by DSDmark** [reply]
I guess, for your last option, you can create a support ticket on the [GitHub website](https://support.github.com/request/education?tags=hubberfy_education). In your ticket, explain your situation and request that your account be changed from a student account to a teacher account.

**Reply by zohim** [reply]
Done, Thanks

**Comment by dairaine** [comment]
Hi, I am in the same situation : I  already completed the approval process, but instead of choosing teacher option, I selected the student. I just need to be reversed to the teacher status. GitHub does not allow a second approval submission with my ID.

**Comment by wahlquisty** [comment]
I am also in the same position. I did not know there was a teacher's package, so I updated my student account for another year. I can't seem to be able to update my account to Teacher.
